Implement Project Card

houstongunter opened the pull request:
- Added two project card components
- One component has the text left of the image
- Other component has the text right of the image
- Closes #10 
![Image 7-5-24 at 12 02 PM](https://github.com/acm-ucr/aiatucr-website/assets/147109020/0f52d74a-44e3-45bf-8b7f-e9ccfb2ff044)

khandrew1 commented on src/components/landing/ProjectCardLeft.js:
```diff
@@ -0,0 +1,36 @@
+const ProjectCardLeft = ({ projectName, description }) => {
+  return (
+    <div className="flex flex-col-reverse md:flex-row items-center md:items-start">
+      {/* Text below the image on small screens */}
```
remove all comments

khandrew1 commented on src/components/landing/ProjectCardLeft.js:
```diff
@@ -0,0 +1,36 @@
+        <div className="absolute inset-0 flex items-center justify-center">
+          <p
+            className="text-ai-blue-500 text-center"
+            style={{ fontSize: "clamp(1rem, 5vw, 2rem)" }}
```
no normal css, you shouldnt even need to do something this complicated?

khandrew1 commented on src/components/landing/ProjectCardRight.js:
```diff
@@ -0,0 +1,33 @@
+        <div className="absolute inset-0 flex items-center justify-center">
+          <p
+            className="text-ai-blue-500 text-center"
+            style={{ fontSize: "clamp(1rem, 5vw, 2rem)" }}
```
no normal css again

khandrew1 commented on src/components/landing/ProjectCardRight.js:
```diff
@@ -0,0 +1,33 @@
+          alt=""
+          width={833}
+          height={436.31}
+          className="w-full h-auto"
```
sorry for being confusing on this in the beginning, please import `projectCard.webp` on top like
`import projectCard from "@/public/projectCard.webp"` and then use that as the source and remove width and height

khandrew1 commented on src/components/landing/ProjectCardLeft.js:
```diff
@@ -0,0 +1,36 @@
+          width={833}
+          height={436.31}
+          className="w-full h-auto"
+        />
```
sorry for being confusing on this in the beginning, please import `projectCard.webp` on top like
`import projectCard from "@/public/projectCard.webp"` and then use that as the source and remove width and height

khandrew1 commented:
Component files should be using `.jsx` file extension, also should be in the folder `components/projects`

khandrew1 commented on src/app/page.js:
```diff
@@ -1,7 +1,20 @@
-import OfferingCards from "@/components/landing/OfferingCards";
+import React from "react";
```
remove react imports from every file

khandrew1 commented on src/app/page.js:
```diff
@@ -1,7 +1,20 @@
+import ProjectCardRight from "@/components/projects/ProjectCardRight";
+import ProjectCardLeft from "@/components/projects/ProjectCardLeft";
 
 const Home = () => {
```
change this back to only offering cards, put these project cards on /projects

khandrew1 commented on src/app/projects/page.js:
```diff
@@ -1,3 +1,5 @@
+import React from "react";
+
 const Page = () => {
   return <>Projects</>;
```
put project card components on this page

khandrew1 commented on src/components/projects/ProjectCardLeft.jsx:
```diff
@@ -0,0 +1,31 @@
+        <Image
+          src="/images/projectCard.webp"
+          alt=""
+          width={833}
+          height={436.31}
```
import image statically on top, `import projectCard from @/public/projectCard.webp` then use that as the src, `src={projectCard}` and remove static heights and width

houstongunter commented:
I don't know why one of the tests failed and I don't know why it's saying I added the offering cards files and changed the location of information.jsx I swear I didn't touch those

khandrew1 commented:
don't worry smth messed up within the directories

khandrew1 commented on src/components/projects/OfferingCards.js:
```diff
@@ -0,0 +1,54 @@
+import { GiPodium } from "react-icons/gi";
+import { LiaNewspaperSolid } from "react-icons/lia";
+
+const Offerings = [
+  {
+    title: "SEMINARS",
+    description:
+      "In-depth seminars offering insight into AI's latest technologies and advancements.",
+    icon: <FaUsers className="text-5xl text-ai-blue-500" />,
+  },
+  {
+    title: "GUEST SPEAKERS",
+    description:
+      "Engaging sessions with AI experts, industry professionals, and prominent researchers.",
+    icon: <RiPresentationFill className="text-5xl text-ai-blue-500" />,
+  },
+  {
+    title: "COMPETITIONS",
+    description:
+      "Challenge yourself and team up with peers in our AI competitions. A chance to put theory into practice.",
+    icon: <GiPodium className="text-5xl text-ai-blue-500" />,
+  },
+  {
+    title: "AI NEWS AND OPPORTUNITIES",
+    description:
+      "Stay informed with the most recent developments, breakthroughs, and job opportunities in the AI realm.",
+    icon: <LiaNewspaperSolid className="text-5xl text-ai-blue-500" />,
+  },
+];
```
move this to a data file

khandrew1 replied:
also lowercase variables for data vars
